contents/sponsor.mdx

```diff
@@ -7,47 +10,101 @@
-        <td className="tw:py-2 tw:md:py-4 tw:text-center"><svg width="20" height="20" md:width="26" md:height="26" className="tw:inline-block" viewBox="0 0 122.88 122.87"><path d="m32.82 51.34 14.99-.2 1.12.29a57.69 57.69 0 0 1 8.54 5.99c1.92 1.63 3.76 3.4 5.5 5.32 5.38-8.65 11.11-16.6 17.16-23.9 6.63-8 13.66-15.27 21.05-21.9l1.46-.56H119l-3.3 3.66c-10.13 11.26-19.33 22.9-27.64 34.9A341.166 341.166 0 0 0 65.7 92.13l-2.06 3.97-1.89-4.04c-3.49-7.48-7.66-14.35-12.64-20.49-4.98-6.14-10.77-11.59-17.52-16.22l1.23-4.01z" fill="#99a1af"/><path d="M61.44 0c9.53 0 18.55 2.17 26.61 6.04-3.3 2.61-6.36 5.11-9.21 7.53a50.868 50.868 0 0 0-17.39-3.05c-14.06 0-26.79 5.7-36 14.92-9.21 9.22-14.92 21.94-14.92 36 0 14.06 5.7 26.78 14.92 36s21.94 14.92 36 14.92c14.06 0 26.79-5.7 36-14.92 9.22-9.22 14.91-21.94 14.91-36 0-3.34-.32-6.62-.94-9.78 2.64-3.44 5.35-6.88 8.11-10.28a61.359 61.359 0 0 1 3.35 20.06c0 16.96-6.88 32.33-17.99 43.44-11.12 11.12-26.48 18-43.44 18-16.96 0-32.32-6.88-43.44-18C6.88 93.76 0 78.4 0 61.44 0 44.47 6.88 29.11 17.99 18 29.11 6.88 44.47 0 61.44 0z" fill="#99a1af"/></svg></td>
-        <td className="tw:py-2 tw:md:py-4 tw:text-center"><svg width="20" height="20" md:width="26" md:height="26" className="tw:inline-block" viewBox="0 0 122.88 122.87"><path d="m32.82 51.34 14.99-.2 1.12.29a57.69 57.69 0 0 1 8.54 5.99c1.92 1.63 3.76 3.4 5.5 5.32 5.38-8.65 11.11-16.6 17.16-23.9 6.63-8 13.66-15.27 21.05-21.9l1.46-.56H119l-3.3 3.66c-10.13 11.26-19.33 22.9-27.64 34.9A341.166 341.166 0 0 0 65.7 92.13l-2.06 3.97-1.89-4.04c-3.49-7.48-7.66-14.35-12.64-20.49-4.98-6.14-10.77-11.59-17.52-16.22l1.23-4.01z" fill="#00bcff"/><path d="M61.44 0c9.53 0 18.55 2.17 26.61 6.04-3.3 2.61-6.36 5.11-9.21 7.53a50.868 50.868 0 0 0-17.39-3.05c-14.06 0-26.79 5.7-36 14.92-9.21 9.22-14.92 21.94-14.92 36 0 14.06 5.7 26.78 14.92 36s21.94 14.92 36 14.92c14.06 0 26.79-5.7 36-14.92 9.22-9.22 14.91-21.94 14.91-36 0-3.34-.32-6.62-.94-9.78 2.64-3.44 5.35-6.88 8.11-10.28a61.359 61.359 0 0 1 3.35 20.06c0 16.96-6.88 32.33-17.99 43.44-11.12 11.12-26.48 18-43.44 18-16.96 0-32.32-6.88-43.44-18C6.88 93.76 0 78.4 0 61.44 0 44.47 6.88 29.11 17.99 18 29.11 6.88 44.47 0 61.44 0z" fill="#00bcff"/></svg></td>
+        <th scope="row" className="tw:px-3 tw:md:px-5 tw:py-3 tw:md:py-4 tw:text-[17px] tw:md:text-[21px] tw:tracking-wide tw:xl:whitespace-nowrap tw:w-0 tw:text-left tw:font-semibold tw:text-slate-900">Website exposure (6 months)</th>
+        <td className="tw:py-2 tw:md:py-4 tw:text-center">
+          <FontAwesomeIcon
+            icon={faCircleCheck}
+            className="tw:inline-block tw:text-xl"
+            style={{ color: "#ffb900" }}
+          />
```

Comment:
@jack482653 I follow https://github.com/sciwork/swconf2023/blob/0a2bd693786ebd328f177543ded0ed3da3fba58f/contents/sponsors.mdx?plain=1#L4, that use `FontAwesomeIcon` !